tests/data/axis/convert_axis.py

```diff
@@ -23,6 +23,20 @@ def test_convert_variables():
     data.close()
 
 
+def test_convert_variable():
```

Comment:
I would like a more descriptive name for this test, it is too similar to the test above it (in name), which could cause confusion if/when one of them fails (especially if the other passes)

Reply:
Changed to more descriptive name.